airflow/providers/google/marketing_platform/hooks/analytics.py

```diff
@@ -0,0 +1,88 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+from googleapiclient.discovery import Resource, build
+
+from airflow.providers.google.cloud.hooks.base import CloudBaseHook
+
+
+class GoogleAnalyticsHook(CloudBaseHook):
+    """
+    Hook for Google Analytics 360.
+    """
+
+    def __init__(
+        self,
+        api_version: str = "v3",
+        gcp_connection_id: str = "google cloud default",
+        *args,
+        **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.api_version = api_version
+        self.gcp_connection_is = gcp_connection_id
+        self._conn = None
+
+    def get_conn(self) -> Resource:
+        """
+        Retrieves connection to Google Analytics 360.
+        """
+        if not self._conn:
+            http_authorized = self._authorize()
+            self._conn = build(
+                "analytics",
+                self.api_version,
+                http=http_authorized,
+                cache_discovery=False,
+            )
+        return self._conn
+
+    # pylint: disable=unused-argument
+    def list_accounts(self, max_results: int, start_index: int) -> list:
```

Comment:
```suggestion
    def list_accounts(self, max_results: int, start_index: int) -> List[Dict[str, Any]]:
```
Let's try to avoid generic types ;)

Reply:
Nice :) I will do it.

Reply:
Yes, it's more explicit and mypy can perform better checks

Reply:
Got it.